File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/ContentService.java
package com.pinyougou.sellergoods.service;
import java.util.List;
import com.pinyougou.pojo.TbContent;							  
import entity.PageResult;
import entity.Result;
/**
 * content服务层接口
 * @author jieweili
 *
 */
public interface ContentService {

    /**
     * 返回全部列表(已弃置)
     * @return
     */
    List<TbContent> findAll();
 
    /**
     * 返回分页对象(过时)
     * @param pageNum
     * @param pageSize
     * @return
     */
	 
    PageResult findPage(Integer pageNum, Integer pageSize);

    /**
     * 添加数据
     * @param content
     * @return
     */
    public Result saveContent(TbContent content);

    /**
     * 根据id查询一个对象
     * @param id
     * @return
     */
    TbContent getContentById(Long id);

    /**
     * 修改对象
     * @param content
     * @return
     */
    Result updateContent(TbContent content);

    /**
     * 批量删除
     * @param idList
     * @return
     */
    Result removeContent(Long[] idList);

    /**
     * 根据条件分页查询
     * @param content
     * @param pageNum
     * @param pageSize
     * @return
     */
    PageResult listContentByExample(TbContent content, Integer pageNum, Integer pageSize);
 
}

File: pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/contentService.js
app.service('contentService', function ($http) {
    //旧的查询方法(弃置)								   
    this.findPage = function (pageNum, pageSize) {
        return $http.get('../content/findPage.do?pageNum=' + pageNum + '&pageSize=' + pageSize)
    };

    //查询方法,读取列表数据绑定到表单中
    this.search = function (pageNum, pageSize, searchEntity) {
        return $http.post('../content/findPage.do?pageNum=' + pageNum + "&pageSize=" + pageSize, searchEntity)
    };

    //删除方法
    this.removeContentList = function (idList) {
        return $http.get('../content/removeContent.do?idList=' + idList)
    };

    //根据id查询
    this.findOne = function (id) {
        return $http.get('../content/findOne.do?id=' + id)
    };

    //修改品牌
    this.updateContent = function (content) {
        return $http.post('../content/updateContent.do', content)
    };

    //新增品牌
    this.saveContent = function (content) {
        return $http.post('../content/saveContent.do', content)
    };   
});
